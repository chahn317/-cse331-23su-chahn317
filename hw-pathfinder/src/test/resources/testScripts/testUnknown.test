# Test case where either start or dest doesn't exist in the graph

CreateGraph graph1

AddNode graph1 A

# dest doesn't exist

FindPath graph1 A B

# start doesn't exist

FindPath graph1 C A