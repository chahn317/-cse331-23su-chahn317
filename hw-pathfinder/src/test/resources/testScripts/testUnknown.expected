# Test case where either start or dest doesn't exist in the graph

created graph graph1

added node A to graph1

# dest doesn't exist

unknown: B

# start doesn't exist

unknown: C